### PR TITLE
Update docs with information useful to reporters

### PR DESCRIPTION
**Pull request by kruai**
## Overview

This PR is being created to address [RHCLOUD-13259](https://issues.redhat.com/browse/RHCLOUD-13259).
It just adds a couple of sections to our docs detailing ways to debug validation issues.

## PR Checklist

- [x] Keep PR title short, ideally under 72 characters
- [ ] Descriptive comments provided in complex code blocks
- [ ] Tests: validate optimal/expected output
- [ ] Tests: validate exceptions and failure scenarios
- [ ] Tests: edge cases
- [x] Documentation, if this PR changes the way other services interact with host inventory
- [ ] Links to related PRs

## Secure Coding Practices Checklist GitHub Link

- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist

- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: docs/_index.md [hunk 1]
```diff
@@ -175,6 +175,31 @@ Multiple tools are set up to help with analysis of rejected messages:
...
 
+## Message Validation Tools
+
+### Host-Ingress Stats Table
+
+HBI's Grafana dashboard keeps track of multiple metrics regarding host-ingress validation.
+For each reporter, [this panel](https://grafana.app-sre.devshift.net/d/EiIhtC0Wa/inventory?viewPanel=68&orgId=1&refresh=5m&var-datasource=crcp01ue1-prometheus&var-kibana=https:%2F%2Fkibana.apps.crcp01ue1.o9m8.p1.openshiftapps.com%2Fapp%2Fkibana%23%2Fdiscover%3F_g%3D(refreshInterval:(pause:!t,value:0),time:(from:now-12h,mode:quick,to:now))%26_a%3D(columns:!(_source),index:%270a2575c0-65f9-11e9-ba88-097c6c9f6850%27,interval:auto&var-kibana_dev=https:%2F%2Fkibana-kibana.5a9f.insights-dev.openshiftapps.com%2Fapp%2Fkibana%23%2Fdiscover%3F_g%3D(refreshInterval:(pause:!t,value:0),time:(from:now-12h,mode:quick,to:now))%26_a%3D(columns:!(_source),index:d2da0310-7048-11e9-a501-7998195d35a8,interval:auto&var-kibana_prod=https:%2F%2Fkibana.apps.crcp01ue1.o9m8.p1.openshiftapps.com%2Fapp%2Fkibana%23%2Fdiscover%3F_g%3D(refreshInterval:(pause:!t,value:0),time:(from:now-12h,mode:quick,to:now))%26_a%3D(columns:!(_source),index:%270a2575c0-65f9-11e9-ba88-097c6c9f6850%27,interval:auto)
```
Comment (gmcculloug) [inline docs/_index.md:183]: The link is not displaying correctly, you need to escape all the `(` and `)` characters in the URL.  😦 

Here's the link with the characters escaped:
`[this panel](https://grafana.app-sre.devshift.net/d/EiIhtC0Wa/inventory?viewPanel=68&orgId=1&refresh=5m&var-datasource=crcp01ue1-prometheus&var-kibana=https:%2F%2Fkibana.apps.crcp01ue1.o9m8.p1.openshiftapps.com%2Fapp%2Fkibana%23%2Fdiscover%3F_g%3D\(refreshInterval:\(pause:!t,value:0\),time:\(from:now-12h,mode:quick,to:now\)\)%26_a%3D\(columns:!\(_source\),index:%270a2575c0-65f9-11e9-ba88-097c6c9f6850%27,interval:auto&var-kibana_dev=https:%2F%2Fkibana-kibana.5a9f.insights-dev.openshiftapps.com%2Fapp%2Fkibana%23%2Fdiscover%3F_g%3D\(refreshInterval:\(pause:!t,value:0\),time:\(from:now-12h,mode:quick,to:now\)\)%26_a%3D\(columns:!\(_source\),index:d2da0310-7048-11e9-a501-7998195d35a8,interval:auto&var-kibana_prod=https:%2F%2Fkibana.apps.crcp01ue1.o9m8.p1.openshiftapps.com%2Fapp%2Fkibana%23%2Fdiscover%3F_g%3D\(refreshInterval:\(pause:!t,value:0\),time:\(from:now-12h,mode:quick,to:now\)\)%26_a%3D\(columns:!\(_source\),index:%270a2575c0-65f9-11e9-ba88-097c6c9f6850%27,interval:auto)`

File: docs/_index.md [hunk 2]
```diff
@@ -175,6 +175,31 @@ Multiple tools are set up to help with analysis of rejected messages:
...
+an inventory exception ("inventory errors"), or an unhandled failure ("failures").
+Each link in the table leads to a reporter-specific Kibana query that
+returns a detailed list of errors that can help with debugging.
+
+### System Profile Validator
+
+The System Profile Validator is a bot that validates changes to the
+System Profile against Production data. To see how a schema change
+would affect current data, open a PR to the [inventory-schemas repo](https://github.com/RedHatInsights/inventory-schemas)
+with your desired modifications. `dippy-bot` will then comment on the PR
```
Comment (gmcculloug) [inline docs/_index.md:194]: Since this is the first reference to `dippy-bot` I would reword this to say something like:
`A bot account (dippy-bot) will then comment...`

File: docs/_index.md (hunk 1 above)
Comment (kruai) [reply docs/_index.md:183]: Ah, VS Code's preview window might have tried to figure it out despite the missing escapes. Thanks!
